app/models/service.rb

```diff
@@ -218,10 +218,6 @@ def composite?
     children.present?
   end
 
-  def retireable?
-    type.present?
-  end
```

Comment:
This method is now included from the [CiFeatureMixin](https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/ci_feature_mixin.rb#L2) which always returns `true`.  👍

Reply:
The rational for changing this check is that, while the blanket check for type being present has some logical issues since not all services have types but all are retireable, the part that's being omitted at the moment is subservices. Per https://github.com/ManageIQ/manageiq/pull/17317#discussion_r186528878, I think checking to see if the service is a subservice first makes more sense.